src/atn-editor/store/reducers/metaReducer.ts

```diff
@@ -0,0 +1,20 @@
+import { AnyAction } from 'redux';
+
+export interface StateType {
```

Comment:
I think that name is super generic. Maybe that one would be better?
```suggestion
export interface MetaReducerState {
```